docs/integrations/query.mdx

```diff
@@ -240,3 +240,66 @@ FIXME: Update demo
 FIXME: Update demo
 
 <CodeSandbox id="u4sli" />
+
+## Devtools
+
+In order to use the Devtools, you need to install it additionally.
+
+```bash
+$ npm i @tanstack/react-query-devtools
+# or
+$ pnpm add @tanstack/react-query-devtools
+# or
+$ yarn add @tanstack/react-query-devtools
+```
+
+All you have to do is put the `<ReactQueryDevtools />` in the `<QueryClientProvider />`.
+
+```tsx
+import { QueryClientProvider } from '@tanstack/react-query'
+import { ReactQueryDevtools } from '@tanstack/react-query-devtools'
+import { queryClientAtom } from 'jotai-tanstack-query'
+
+export const App = () => {
+  const queryClient = useAtomValue(queryClientAtom)
+
+  return (
+    <QueryClientProvider client={queryClient}>
+      <ReactQueryDevtools />
+      <Provider>
+        <App />
+      </Provider>
```

Comment:
```suggestion
      <App />
```
I think omitting Provider here makes things more consistent.
But, I'm actually okay to drop the first example and only leave the second one.